### DISCUSSION TITLE
I Lost 2FA github

### DISCUSSION THREAD

**Discussion by joaopa24** [community/community#118970] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I recently lost access to my github due to the google authentication app, I received an email from git itself asking me to enable 2FA, because I certainly wouldn't enable that shit on my own, and I did it with the google app that Github itself recommended it, it turned out that the application didn't store the codes in the cloud, now I'm here based on what I read, they won't help me at all, what can I do? Is there any way for me to prove that the account is mine, the photo on the account is a photo of me, I added it years ago and didn't change it, I logged into the account in a specific city that is my hometown and my college. Unfortunately I don't have another method, my old computer was logged into github but when I set it up, it was already disconnected, I can still push repositories through vscode, does that mean I'm authenticated?

**Comment by ettaboyle** [comment, marked as answer]
Hi @joknz21! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
